### PR TITLE
Add custom-inherit 2.2.2

### PR DESCRIPTION
**Pull request by rsokl**
Checklist

- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/master/recipes/example/meta.yaml#L57-L66) for an example)
- [x] Source is from official source
- [x] Package does not vend other packages
- [x] Build number is 0
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details)
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there

### CODE REVIEW

**Comment by rsokl** [comment]
@Zeitsperre I switched branches for the new version

File: recipes/custom-inherit/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+{% set version = "2.2.2" %}
+
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: "https://pypi.io/packages/source/c/{{ name }}/custom_inherit-{{ version }}.tar.gz"
+  sha256: "83c048bc3415a9e38e44e78dbe231f837aa3d4fd91b4e71443b6f6e38034f583"
```
Comment (rsokl) [inline recipes/custom-inherit/meta.yaml]:
```suggestion
  sha256: 83c048bc3415a9e38e44e78dbe231f837aa3d4fd91b4e71443b6f6e38034f583
```

**Comment by rsokl** [comment]
@conda-forge/staged-recipes it looks like there was [an error spinning up the linux build](https://dev.azure.com/conda-forge/feedstock-builds/_build/results?buildId=81085&view=logs&jobId=10bda42a-d420-5c28-a43f-bc4e66405e5c&taskId=bef775d0-a1a2-55a3-9721-17147029baf0&lineStart=60&lineEnd=62&colStart=1&colEnd=25):

>Unable to find image 'condaforge/linux-anvil-comp7:latest' locally
docker: Error response from daemon: received unexpected HTTP status: 503 Service Unavailable.
See 'docker run --help'.

Could you trigger a rebuild? Also this should be ready for review

**Comment by rsokl** [comment]
@Zeitsperre any idea why I can't seem to "@" any of the help-teams? My tags don't seem to link to them..

**Comment by CJ-Wright** [comment]
docker is down https://status.docker.com/pages/533c6539221ae15e3f000031

**Comment by rsokl** [comment]
Should we ping you when it is back up? To re-run the build?

**Comment by CJ-Wright** [comment]
I've restarted the failing build.

**Comment by rsokl** [comment]
Thank you! Looks like we are ready for review :)

File: recipes/custom-inherit/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: "https://pypi.io/packages/source/c/{{ name }}/custom_inherit-{{ version }}.tar.gz"
+  sha256: 83c048bc3415a9e38e44e78dbe231f837aa3d4fd91b4e71443b6f6e38034f583
+
+build:
+  number: 0
+  script: "{{ PYTHON }} -m pip install . --no-deps --ignore-installed -vv "
```
Comment (CJ-Wright) [inline recipes/custom-inherit/meta.yaml]: Can this be noarch? https://conda-forge.org/docs/maintainer/knowledge_base.html#noarch-builds

File: recipes/custom-inherit/meta.yaml [hunk 3]
```diff
@@ -0,0 +1,39 @@
...
+    - custom_inherit
+
+about:
+  home: "https://github.com/rsokl/custom_inherit"
+  license: "MIT"
+  license_family: "MIT"
+  license_file: "LICENSE.md"
+  summary: "A Python package that provides customized docstring inheritance schemes between derived classes and their parents."
+  doc_url: "https://github.com/rsokl/custom_inherit/blob/master/README.md"
+  dev_url: ""
```
Comment (CJ-Wright) [inline recipes/custom-inherit/meta.yaml]: I would remove this, rather than having an empty string

File: recipes/custom-inherit/meta.yaml (hunk 2 above)
Comment (rsokl) [reply recipes/custom-inherit/meta.yaml]: Yes, it looks like this. Would this be correct syntax?
```suggestion
  noarch: python
  script: "{{ PYTHON }} -m pip install . --no-deps --ignore-installed -vv "
```

File: recipes/custom-inherit/meta.yaml (hunk 3 above)
Comment (rsokl) [reply recipes/custom-inherit/meta.yaml]:
```suggestion

```

File: recipes/custom-inherit/meta.yaml (hunk 2 above)
Comment (CJ-Wright) [reply recipes/custom-inherit/meta.yaml]: yep!

File: recipes/custom-inherit/meta.yaml (hunk 3 above)
Comment (CJ-Wright) [reply recipes/custom-inherit/meta.yaml]: this looks good too

**Comment by CJ-Wright** [comment]
This is noarch and so only needs to pass on linux.

**Comment by CJ-Wright** [comment]
Thank you very much!

**Comment by rsokl** [comment]
Thanks! What, roughly, is the waiting period I should expect before `conda install -c conda-forge custom-inherit` will result in the package being found?

**Comment by rsokl** [comment]
@CJ-Wright I am not sure what the implications of this is, but [CircleCI had a build error](https://circleci.com/gh/conda-forge/custom-inherit-feedstock/1?utm_campaign=vcs-integration-link&utm_medium=referral&utm_source=github-build-link) on the custom-inherit-feedstock. 

>#!/bin/sh -eo pipefail
>No configuration was found in your project. Please refer to https://circleci.com/docs/2.0/ to get started with your configuration.

Is this why `PackagesNotFoundError` is popping up when I try out the installation

**Comment by CJ-Wright** [comment]
the package takes about 20 mins after it's first build to be indexed. we can follow up on the feedstock, I've asked for a rerender

**Comment by CJ-Wright** [comment]
@rsokl do you have conda-forge in your .condarc?

**Comment by rsokl** [comment]
Locally? I do not. I was manually specifying `-c conda-forge` when trying the install

**Comment by CJ-Wright** [comment]
That should work too. Can you try again and post the contents of your `conda info` and `conda list`?

**Comment by rsokl** [comment]
Will do when I get home. Thanks for the help! I can also open an issue on the custom-inherit feedstock if that's better

**Comment by CJ-Wright** [comment]
It seems that the package hasn't made it on to the mirror yet. You can check here: https://conda-static.anaconda.org/conda-forge/rss.xml (for instance pip was merged 5 hours ago).

**Comment by rsokl** [comment]
It looks like the package is on the mirror. Thanks again for the help!

**Pull request merged**